docs/examples/advanced-cue/app1.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: Application
+metadata:
+  name: testapp
+spec:
+  components:
+    - name: express-server
+      settings:
+        cmd:
+          - node
+          - server.js
+        image: oamdev/testapp:v1
+        port: 8080
+      traits:
+        - name: new-route
+          properties:
+            domain: test.my.domain
+            http:
+              "/api": 8080
+      type: webservice
```

Comment:
why `type` comes here?

Reply:
Yes， it's required value in Application object.

Reply:
I mean why this field comes to the end?